raml-maven-plugin/src/test/resources/lint-check/src/raml/cakeshop-command-api.raml

```diff
@@ -0,0 +1,22 @@
+#%RAML 0.8
```

Comment:
Let's simplify this RAML (and JSON if its actually needed) to the bare minimum to its clear that we aren't testing the content.